packages/components/src/Popover/Popover.tsx

```diff
@@ -46,15 +47,39 @@ Popover.Arrow = function PopoverArrow({
   UNSAFE_className,
   UNSAFE_style,
 }: PopoverArrowProps) {
-  const { setArrowElement, popperStyles } = usePopoverContext();
+  const { setArrowElement, popperStyles, placement } = usePopoverContext();
   const popoverStyles = usePopoverStyles();
   const classes = classnames(popoverStyles.arrow, UNSAFE_className?.arrow);
 
+  // the arrow will get positioned opposite to the placement side
+  const staticSideMap: Record<Side, Side> = {
+    top: "bottom",
+    right: "left",
+    bottom: "top",
+    left: "right",
+  } as const;
+
+  // placement can include alignment (e.g. 'top-start') but we just want the side
+  const placementKey = placement?.split("-")[0] as Side;
+  const staticSide = staticSideMap[placementKey];
+
+  const arrowStyles: React.CSSProperties = {
+    position: "absolute",
+    // only left or top will be defined at a time
+    left: popperStyles.arrow?.x != null ? `${popperStyles.arrow?.x}px` : "",
+    top: popperStyles.arrow?.y != null ? `${popperStyles.arrow?.y}px` : "",
+    right: "",
+    bottom: "",
+    [staticSide]: "-7px",
```

Comment:
I forget, what's this correspond to? the width of the base of the triangle for the arrow?

Reply:
oh no, it's for placing it on the edge of the popover. hmm I don't get how we get to `-7` when the width/height is 16px

if you understand it, plopping that value into a variable with a good name would make this easier to read.

Reply:
gotcha. ok nice, I like us referring to that variable much better!